cfgmgr/vrfmgr.cpp

```diff
@@ -273,3 +291,170 @@ void VrfMgr::doTask(Consumer &consumer)
         it = consumer.m_toSync.erase(it);
     }
 }
+
+bool VrfMgr::doVrfEvpnNvoAddTask(const KeyOpFieldsValuesTuple & t)
+{
+    SWSS_LOG_ENTER();
+    string tunnel_name = "";
+    const vector<FieldValueTuple>& data = kfvFieldsValues(t);
+    for (auto idx : data)
+    {
+        const auto &field = fvField(idx);
+        const auto &value = fvValue(idx);
+
+        if (field == "source_vtep")
+        {
+            tunnel_name = value;
+        }
+    }
+
+    if (m_evpnVxlanTunnel.empty())
+    {
+        m_evpnVxlanTunnel = tunnel_name;
+        VrfVxlanTableSync(true);
+    }
+
+    SWSS_LOG_NOTICE("Added evpn nvo tunnel %s", m_evpnVxlanTunnel.c_str());
+    return true;
+}
+
+bool VrfMgr::doVrfEvpnNvoDelTask(const KeyOpFieldsValuesTuple & t)
+{
+    SWSS_LOG_ENTER();
+
+    if (!m_evpnVxlanTunnel.empty())
+    {
+        VrfVxlanTableSync(false);
+        m_evpnVxlanTunnel = "";
+    }
+
+    SWSS_LOG_NOTICE("Removed evpn nvo tunnel %s", m_evpnVxlanTunnel.c_str());
```

Comment:
If m_evpnVxlanTunnel is empty, this log wouldnt print anything.

If m_evpnVxlanTunnel is not empty
   m_evpnVxlanTunnel will be empty  after m_evpnVxlanTunnel = "";

Finally this log wouldnt print anything for m_evpnVxlanTunnel ?